v1/package.json

```diff
@@ -0,0 +1,64 @@
+{
+  "name": "distributed-press-api",
+  "version": "0.1.0",
+  "description": "API for the Distributed Press",
+  "main": "build/index.js",
+  "scripts": {
+    "clean": "rm -rf build",
+    "build": "npm run clean && tsc",
+    "fmt": "ts-standard --fix",
+    "dev": "npm run build && node build/index.js | pino-pretty -c -t",
+    "start": "node build/index.js",
+    "test": "ava"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/hyphacoop/api.distributed.press.git"
+  },
+  "keywords": [
+    "publish",
+    "dweb",
+    "hypercore",
+    "dat",
+    "ipfs",
+    "web monetization",
+    "ethereum"
+  ],
+  "author": "Distributed Press",
```

Comment:
I think the author should be HyphaCoop (or similar) cc @benhylau thoughts?

Reply:
Just copied this from the old `package.json` but happy to change!

Reply:
Might as well keep what was there already then!